src/Core/Domain/Cart/QueryResult/CartInformation/CartProduct.php

```diff
@@ -197,4 +205,12 @@ public function getCustomization(): ?Customization
     {
         return $this->customization;
     }
+
+    /**
+     * @return bool
+     */
+    public function isGift(): bool
```

Comment:
I don't see any use off this getter, and I also wonder where in the template or the javascript is this data injected?

Reply:
most of this class getters have no usage. I generated all of them i think, its basically a DTO which is used (after converting to json) in javascript.
I think json serializing would fail due to private properties without getters behind the scenes.